code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -245,6 +247,30 @@
 	item_state = "luckpacket"
 	item_obj = /obj/item/clothing/mask/smokable/cigarette/lucky
 
+/obj/item/storage/fancy/cigarettes/robusto
+	name = "\improper Red Robusto packet"
+	desc = "A packet of six Red Robusto Cigarettes. Old NanoTrasen cigs, some say they make you more robust..."
```

Comment:
```suggestion
	desc = "A packet of six Red Robusto Cigarettes. Old Nanotrasen cigs, some say they make you more robust..."
```

Reply:
pretty sure the t is capitalized in most applications

Reply:
It is both referred as NanoTrasen and Nanotrasen.
It should be Nanotrasen.